README.md

```diff
@@ -64,8 +72,8 @@ Your Answer:
 
 Select all that apply:
 ```
-[] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing.
-[] Synchronous code runs all at the same time, asynchronous code runs completely randomly
+[x] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing.
+[x] Synchronous code runs all at the same time, asynchronous code runs completely randomly
 [] Synchronous code runs in order (as appears in the source), asynchronous code may run at a later time.
 ```
```

Comment:
was only able to briefly go over asynch vs synch in the events callbacks class, might be enough to leave this question out until week 7